File: code/TemplateStudioForUWP/Templates/Uwp/Features/BackgroundTask._VB/Param_ProjectName/Services/BackgroundTaskService$BackgroundTaskFeature_gpostaction.vb
'{**
' This code block adds the BackgroundTaskFeature to the method `CreateInstances()` of the BackgroundTaskService of your project.
'**}

Namespace Services
    Friend Class BackgroundTaskService
        Inherits ActivationHandler(Of BackgroundActivatedEventArgs)

        Private Shared Function CreateInstances() As IEnumerable(Of BackgroundTask)
            Dim backgroundTasks As List(Of BackgroundTask) = New List(Of BackgroundTask)()
            '^^
            '{[{
            backgroundTasks.Add(New BackgroundTaskFeature())
            '}]}
            Return backgroundTasks
        End Function
    End Class
End Namespace

File: code/TemplateStudioForUWP/Templates/Uwp/Features/BackgroundTask/Param_ProjectName/Services/BackgroundTaskService$BackgroundTaskFeature_gpostaction.cs
//{**
// This code block adds the BackgroundTaskFeature to the method `CreateInstances()` of the BackgroundTaskService of your project.
//**}
namespace Param_RootNamespace.Services
{
    internal class BackgroundTaskService : ActivationHandler<BackgroundActivatedEventArgs>
    {
        private static IEnumerable<BackgroundTask> CreateInstances()
        {
            var backgroundTasks = new List<BackgroundTask>();
//^^
//{[{
            backgroundTasks.Add(new BackgroundTaskFeature());
//}]}
            return backgroundTasks;
        }
    }
}
